EIPS/eip-5252.md

```diff
@@ -0,0 +1,53 @@
+---
+eip: 5252
+title: Account-bound Finance
+description: A smart contract design pattern in decentralized finance to remove insolvency and arbitrary loss of funds using blockchain technology only. Extension of EIP-5114
+author: Hyungsuk Kang (@hskang9)
+discussions-to: https://ethereum-magicians.org/t/
+type: Standards Track
+category: ERC
+status: Draft
+requires: 721, 1155, 5114
+created: 2022-06-29
+---
+
+## Abstract
+This EIP proposes a form of smart contract design pattern and a new type of account abstraction on how one's finance should be managed in web3.0, ensuring transparency of managing investments and protection with self-sovereignty even from its financial operators.
+
+This EIP also extends account-bound token from [EIP-5114](https://eips.ethereum.org/EIPS/eip-5114), but the difference is that it can be transferred to other accounts for mobility between managing multiple wallets.
```

Comment:
```suggestion
This EIP also extends [EIP-5114](./eip-5114.md), but the difference is that it can be transferred to other accounts for mobility between managing multiple wallets.
```
Should be an internal link.